Revert/Sources/PickerTableViewController.swift

```diff
@@ -0,0 +1,68 @@
+// Copyright © 2020 Itty Bitty Apps. All rights reserved.
+
+import UIKit
+
+final class PickerTableViewController: UITableViewController {
+  private let sections: [CountrySection] = RevertItems.capitalCities.data()
+  private let datePickerCellIdentifier = "DatePickerCell"
+
+  private enum Section: Int, CaseIterable {
+    case datePicker = 0
+    case other = 1
+  }
+
+  override func viewDidLoad() {
+    super.viewDidLoad()
+
+    tableView.register(UINib(nibName: "DatePickerTableViewCell", bundle: nil), forCellReuseIdentifier: datePickerCellIdentifier)
+  }
+
+  override func numberOfSections(in tableView: UITableView) -> Int {
+    return Section.allCases.count
+  }
+
+  override func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+    if #available(iOS 14.0, *) {
+      return section == Section.datePicker.rawValue ? 3 : 1
```

Comment:
Just a minor suggestion here, what do you think of adding this to the class: 
```
 @available(iOS 14, *)
 var datePickerStyles: [UIDatePickerStyle] { [.compact, .inline, .wheels] }
```
then
```suggestion
      return section == Section.datePicker.rawValue ? datePickerStyles.count : 1
```
In case Apple adds new styles to the picker in the future 😉

Reply:
Good idea Decheng 👍